include/nn/nex/berkeley.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <nn/types.h>
+
+namespace nn {
+namespace nex {
+
+namespace TransportProtocol {
+enum Type { unk1, unk2, unk3, unk4, unk5, unk6 };
```

Comment:
There should be a trailing comma after `unk6` so that this is formatted correctly